amr.md

```diff
@@ -2700,36 +2700,33 @@ Annotators are encouraged to use `have-degree-91` as the root concept (as oppose
 > That boy is more bright.
 
 ```lisp
-(b / bright-03
-   :ARG1 (b2 / boy
-               :mod (t / that))
-   :degree (m / most))
+(b / boy 
+      :ARG1-of (h / have-degree-91 
+            :ARG2 (b2 / bright-03 
+                  :ARG1 b) 
+            :ARG3 (m / most))) 
 ```
 
 > That boy is the brightest.
```

Comment:
With `boy` as the focus, shouldn't this be "The brightest boy"?

Reply:
Sure, will update. I updated these according to new consensus sentences, didn't take as much time as probably needed.

Reply:
Actually, I cannot make changes at this point -- all that I can access is the old version. And as mentioned previously, GitHub doesn't allow me to preview these changes before submitting a proposal, so I had to do edits strictly in MD without seeing them finalized.  I apologize for oversights. Can you make changes needed that you've caught here?

Reply:
I think you should be able to modify the pull request. I don't think I can do it myself.

Reply:
Try viewing the file with your changes (https://github.com/cbonial/amr-guidelines/blob/4924e01f279f68e8d688d1c46af9ec63980c83ae/amr.md) and clicking edit.

Reply:
Actually https://github.com/cbonial/amr-guidelines/blob/patch-2/amr.md